README.md

```diff
@@ -79,10 +79,10 @@ Developers are free to choose between npm and yarn for managing the dependencies
 - `yarn test-node` or `npm run test-node` runs the testsuite in NodeJS only.
 
 #### Run ESLint
-`yarn lint` runs the ESLint javascript linter.
+`yarn lint` or `npm run lint` runs the ESLint javascript linter.
```

Comment:
For conistency, I would now list them in reverse order, i.e., npm first, yarn second. This is also the order in which we list them before.